pkg/release/release.go

```diff
@@ -71,24 +75,83 @@ const (
 	WindowsGCSPath = "gcs-stage/extra/gce/windows"
 )
 
+var (
+	DefaultToolOrg = git.DefaultGithubOrg
+)
+
+// GetDefaultKubernetesRepoURL returns the default HTTPS repo URL for Release Engineering tools.
+// Expected: https://github.com/kubernetes/release
+func GetDefaultToolRepoURL() (string, error) {
+	return GetToolRepoURL(DefaultToolOrg, DefaultToolRepo, false)
+}
+
+// GetKubernetesRepoURL takes a GitHub org and repo, and useSSH as a boolean and
+// returns a repo URL for Release Engineering tools.
+// Expected result is one of the following:
+// - https://github.com/<org>/release
+// - git@github.com:<org>/release
+func GetToolRepoURL(org, repo string, useSSH bool) (string, error) {
+	if org == "" {
+		org = GetToolOrg()
+	}
+	if repo == "" {
+		repo = GetToolRepo()
+	}
+
+	return git.GetRepoURL(org, repo, useSSH)
+}
+
+// GetToolOrg checks if the 'TOOL_ORG' environment variable is set.
+// If 'TOOL_ORG' is non-empty, it returns the value. Otherwise, it returns DefaultToolOrg.
+func GetToolOrg() string {
+	toolOrg := os.Getenv("TOOL_ORG")
+	if toolOrg == "" {
+		toolOrg = DefaultToolOrg
+	}
+
+	return toolOrg
+}
+
+// GetToolRepo checks if the 'TOOL_REPO' environment variable is set.
+// If 'TOOL_REPO' is non-empty, it returns the value. Otherwise, it returns DefaultToolRepo.
+func GetToolRepo() string {
+	toolRepo := os.Getenv("TOOL_REPO")
```

Comment:
Big +1 from me about steering awat from using `env` vars directly in functions. The good advice I read some time ago is to use the env vars just in the main function and then pass them around as function parameters.

Reply:
@hasheddan -- I would prefer to keep this one as-is.
The idea being that we want to make it a little difficult to run the tool in a non-`k/release@master` manner.

If you used some flags, it's possible maybe you copy/pasted something.
If you set three environment variables, there's some intent behind override the default tools org, repo, and branch.

Reply:
I definitely see your point, but I could also see someone copy pasting setting `env` vars just like the command with flags. In addition, a flag *must* always be set explicitly on each command run, while an `env` var could be lurking from a previous operation in the same session. All of that being said, I think this is mostly a stylistic preference (rather than a right / wrong thing) so I am comfortable moving forward if you feel strongly :)

Reply:
The case is, it's not about making it harder for people to not use the flags itself, but it's much easier to maintain the thing in the future if logic related to env-vars is stored in the main/root place. These are just good practices and makes code more futureproof.

Reply:
@bartsmykla -- That's essentially what's being done here. 
By putting the logic for these env vars in the `release` package, instead of the command, I'm trying to ensure that anyone who needs to use them has a consistent experience.

As a note, this is one of the first few passes on this tool rewrite.
In this first phase, I'm primarily concerned w/ replicating the logic of the existing tool, which uses env vars:
https://github.com/kubernetes/release/blob/c13fa28c95c0696f2dc6bd4d90d3dacdbf3d1007/gcbmgr#L152-L154

There are some pieces that no longer make sense to carry over, but this isn't one of them just yet.